src/chat/discord.md

```diff
@@ -1,4 +1,104 @@
 ## Discord
 
-We haven't yet written the documentation for Discord, but if you'd like to help
-out, this is the place for it to go!
+Rust's Discord is used by Community, Lang, Ops, and Documentation teams, as well as their working groups. It is also the home of Domain Working Groups and provides a space for general discussion among Rust users, contributors, and beginners. 
+
+# Where to go for help with using Discord
+Discord's support center provides documentation about its [user interface](https://support.discordapp.com/hc/en-us/categories/200404398) and [account settings](https://support.discordapp.com/hc/en-us/categories/200404358).
+
+# Getting started
+
+1) Understand community standards <br/>
+Before joining the conversation on Discord, you can prepare by reading the [Code of Conduct and Moderation Guidelines](https://www.rust-lang.org/community#conduct).
+
+2) Access channels <br/>
+To access the [Rust Discord channel](https://discordapp.com/invite/rust-lang) visit [(https://discordapp.com/invite/rust-lang)](https://discordapp.com/invite/rust-lang) and open the channel in your browser or app. If you do not already have a Discord account, you can register for one as part of the process of gaining access.
```

Comment:
Can you use `https://discord.gg/rust-lang` here? It resolves to the former, but it's more SEO friendly. Also, I think only one link would be best, so people don't feel the need to check two links.

Would also be prudent to mention that they need to agree to our CoC by following the instructions in #welcome.

Reply:
If they are new to the discord they will have to agree to the CoC in the #welcome channel